R/build-search-docs.R

```diff
@@ -75,10 +75,15 @@ url_node <- function(url) {
 #' search:
 #'   exclude: ['news/index.html']
 #' ```
-#' @section Local testing:
-#' Search won't work if you simply use pkgdown preview of the static files.
+#' @section Debugging and local testing:
+#'
+#' Locally (as opposed to on GitHub Pages or Netlify for instance),
+#' search won't work if you simply use pkgdown preview of the static files.
 #' You can use `servr::httw("docs")` instead.
 #'
+#' If search is not working, run `pkgdown::pkgdown_sitrep()` to eliminate
+#' common issues such as the absence of URL in the pkgdown configuration file
```

Comment:
That's the only common issue that function will indicate, but over time it might collect more checks hence the vagueness. But maybe not a good idea to be vague.